packages/amplify-graphql-conversation-transformer/src/__tests__/__snapshots__/amplify-graphql-conversation-transformer.test.ts.snap

```diff
@@ -854,468 +489,3 @@ export function response(ctx) {
   ],
 }
 `;
-
-exports[`ConversationTransformer valid schemas should transform a conversation route with query tools 3`] = `
```

Comment:
A snapshot of the generated GraphQL schema was included for a single test cases. The snapshot assertion is removed from this PR as I didn't see any added value of including it.